data/ui/main_window.ui

```diff
@@ -162,6 +163,7 @@
                 <property name="orientation">vertical</property>
                 <child>
                   <object class="AdwHeaderBar" id="titlebar">
+                    <property name="show-start-title-buttons" bind-source="leaflet" bind-property="folded" bind-flags="sync-create"/>
```

Comment:
This should be show-end-title-buttons.

Reply:
https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.HeaderBar.html It is based from this, I think if I used show-end, it will have some weird bugs when the title-buttons are on the left.

Reply:
Looks to be a typo to me. Also, don't worry about buttons on the left, that's not the defaults on a GNOME app.

Reply:
I just tested it with end, and it lost its title-button when unfolded

Reply:
stitlebar as end, and titlebar as start. Works as expected/similar to headergroup of libhandy

Reply:
I see... then I think it's ok then.